run/action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'run-cmd'
+description: 'Run command'
+inputs:
+  branch:  # id of input
+    description: 'Branch name'
+    required: true
+  command:
+    description: "Command to be run"
+    required: true
+runs:
+  using: "composite"
+  steps:
+      - name: Download repo
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.branch }}
+      - name: Generate script
+        shell: bash
+        # Here it would get the script from previous step
+        run: |
```

Comment:
I believe something like this would work in one line without the need for the file:

```
bash -c "$COMMAND"
```

Reply:
I worked flawlessly! Great recommendation!